### ISSUE TITLE
Inventory of default tags

### ISSUE THREAD

**Issue by stevenbird** [aikuma/aikuma#442]
In this issue I would like to collect a default set of tags for audio recordings:

* [OLAC Discourse Type](http://www.language-archives.org/REC/discourse.html) 10
* Required proficiency for comprehension, cf [ACTFL Proficiency Guidelines](http://en.wikipedia.org/wiki/ACTFL_Proficiency_Guidelines)
* Recording quality
* Audience restriction
* Naturalness, spontaneity?

**Comment by Lingomat** [comment]
https://github.com/aikuma/aikuma/wiki/Default-tags

Assume that we're not talking about 'flags' here, if that's the way we're going.

**Comment by stevenbird** [comment]
Yes, these are tags, not flags (cf #67). Thanks for setting up the wiki page.

**Comment by Lingomat** [comment]
Is this sufficient? It seems like it might be. 

One issue is whether to use simple nouns where feasible or whether to systematically use prefixes. I thought the OLAC discourse types was a good example of something that just makes sense to use as they are.
